cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -170,6 +170,13 @@ public void validateProperties(Map<String, String> properties) {
     if (networkTags.size() > 64) {
       throw new IllegalArgumentException("Exceed Max number of tags. Only Max of 64 allowed. ");
     }
+    //Validate if AutoScaling is enabled and Policy is provided
+    if (conf.isAutoPolicyEnabled()) {
+      if (conf.getAutoScalingPolicy() == null || conf.getAutoScalingPolicy().isEmpty()) {
+        throw new IllegalArgumentException("Autoscaling is enabled but no AutoScaling policy provided. " +
```

Comment:
"Autoscaling is been enabled, but not policy for scaling has been provided"

Reply:
Done